README.md

```diff
@@ -1,26 +1,197 @@
+  <%= csrf_meta_tags %>
+</head>
+<body>
+
+<%= yield %>
+
+</body>
+</html>
+```
+
+## Yield
+
+Let's say you code up a new layout from scratch, and you end up with something like this:
+
+```erb
+<!-- app/views/layouts/application.html.erb -->
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Flatiron Store</title>
+</head>
+<body>
+    <h1>Welcome To The Flatiron Store!</h1>
+</body>
+</html>
+```
+
+**Note:** usually you should include links to assets like style sheets and JavaScript files in your layouts, which was omitted in the code above to keep it simpler.
+
+Other than the missing links to common assets, this layout is missing something **terribly important**. To see what it is, have a look at the following example that uses this layout.
+
+This is an example of an action that we expect to result in our action's template rendered within the layout defined above.
+
+
+```ruby
+#/app/controllers/static_controller.rb
+
+class StaticController < ApplicationController
+  def about
+  end
+end
+```
+
+There should also be an associated route in the /config/routes.rb file to route a request to **/about** to the about action in the static controller above.
+
+```ruby
+#/config/routes.rb
+
+Rails.application.routes.draw do
+  get 'about', to: 'static#about'
+end
+```
+
+And this is the template for the about action, with a simple message in it, which we would want to display in our layout from above.
+
+```erb
+<!-- /app/views/static/about.html.erb -->
+
+<p>Hello!</p>
+```
+
+When you load up this route in your browser, you will be greeted by a very bold message saying: **Welcome To The Flatiron Store!**, but you won't see the **Hello!** from the about action's template.
+
+This is happening because the layout file at *app/views/layouts/application.html.erb* does not have a `yield` statement in it.
+The `yield` keyword is what Rails uses to decide where in the layout to render the content for the action. If you don't put Yield in your layout, only your layout will render for each action, instead of your layout with the contents of the action template in the correct place in the layout.
+
+To fix this issue, just change the layout file to include `yield`:
+
+```erb
+<!-- app/views/layouts/application.html.erb -->
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>ACME Store</title>
+</head>
+<body>
+    <h1>Welcome To The ACME Store!</h1>
+    <%= yield %>    
+</body>
+</html>
+```
+
+Now when you hit up this route in your browser, you will see **Welcome To The ACME Store!**, followed by **Hello!**. This means that when the layout rendered, it pulled the action specific template into the correct place, where we added the `yield` statement.
+
+## How Layouts and Templates Are Stitched Together
+
+At it's simplest level, this is what happens when a request is made to your rails application:
+
+1. Rails finds the template for your action based on either convention, or any other options you passed to the render method in your controller action.
```

Comment:
For each step can you ground it in the concrete and make sure you just mention what file you are talking about from your previous flatiron example